packages/contracts-bedrock/scripts/forges-test-names.ts

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+const testPath = './contracts/test'
+const testFiles = fs.readdirSync(testPath)
+
+const handleFunctionName = (name: string) => {
+  if (!name.startsWith('test')) {
+    return
+  }
+  const parts = name.split('_')
+  parts.forEach((part) => {
+    if (part[0] !== part[0].toLowerCase()) {
+      throw new Error(
+        `Invalid test name: ${name}. Test name parts should be in camelCase`
+      )
+    }
+  })
+  if (parts.length < 3 || parts.length > 4) {
+    throw new Error(
+      `Invalid test name: ${name}. Test names should have either 3 or 4 parts, each separated by underscores`
+    )
+  }
+  if (!['test', 'testFuzz'].includes(parts[0])) {
+    throw new Error(
+      `Invalid test name: ${name}. Names should begin with either "test" or "testFuzz"`
+    )
+  }
+  if (
+    !['succeeds', 'reverts', 'fails', 'differential', 'benchmark'].includes(
+      parts[parts.length - 1]
+    ) &&
+    parts[parts.length - 2] !== 'benchmark'
+  ) {
+    throw new Error(
+      `Invalid test name: ${name}. Test names should end with either "succeeds", "reverts", "fails", "differential" or "benchmark[_num]"`
+    )
+  }
+  if (
+    ['reverts', 'fails'].includes(parts[parts.length - 1]) &&
+    parts.length < 4
+  ) {
+    throw new Error(
+      `Invalid test name: ${name}. Failure tests should have 4 parts. The third part should indicate the reason for failure.`
+    )
+  }
+}
+
+// Todo
```

Comment:
What is this todo?

Reply:
Updated this to say "Todo: Implement a check to validate test contract names"